swagger/control/swagger.yaml

```diff
@@ -0,0 +1,152 @@
+swagger: '2.0'
+info:
+  title: Mailgun Relayery Control API
+  description: |
+    API for managing the authentication tokens and channel settings of remote instances.
+  version: "1.0.0"
+host: mailgun.relayery.control.parquery.com
+schemes:
+  - https
+basePath: /
+tags:
+- name: "control"
+paths:
+  /api/token:
```

Comment:
/api/token and /api/channel are redundant -- you need to set the token in /api/channel anyhow, don't you?